packages/opentelemetry-sdk-node/package.json

```diff
@@ -43,21 +43,21 @@
   },
   "dependencies": {
     "@opentelemetry/api-metrics": "0.25.0",
-    "@opentelemetry/core": "0.25.0",
+    "@opentelemetry/core": "1.0.0",
     "@opentelemetry/instrumentation": "0.25.0",
     "@opentelemetry/resource-detector-aws": "0.24.0",
```

Comment:
Do we really want dependencies to contrib here in the GA version?

Reply:
Hmm no I would like to remove this dependency. IMO vendor-specific resource detectors should not be in the default SDK. I just looked in the spec and it looks like there is no requirement to have it.

/cc @open-telemetry/javascript-maintainers @willarmiros wtyt

Reply:
Yeah you can remove it, I don't believe this is here for other languages. Looks like they were meant to only be including temporarily anyway per description of #1420

Reply:
I think this would be the only blocker here